src/sidebar/util/search-filter.js

```diff
@@ -91,25 +94,25 @@ function tokenize(searchText) {
 export function toObject(searchText) {
   /** @type {Record<string,string[]>} */
   const obj = {};
-  const backendFilter = f => (f === 'tag' ? 'tags' : f);
 
-  const addToObj = (key, data) => {
-    if (obj[key]) {
-      return obj[key].push(data);
-    } else {
-      return (obj[key] = [data]);
-    }
-  };
+  /** @param {string} field */
+  const backendFilter = field => (field === 'tag' ? 'tags' : field);
 
   if (searchText) {
     const terms = tokenize(searchText);
     for (const term of terms) {
-      let [filter, data] = splitTerm(term);
```

Comment:
`term` here is a string like `title:foobar` and `splitTerm` splits it into the field name and value parts. I changed `filter` to `field` as I thought that was a more obvious name.

Reply:
I agree.